test/cli/cli.spec.ts

```diff
@@ -0,0 +1,46 @@
+import tape from 'tape'
+import { spawn } from 'child_process'
+
+tape('[CLI]', (t) => {
+  t.test('should start and begin downloading blocks', { timeout: 160000 }, (t) => {
+    t.plan(1)
+    const file = require.resolve('../../dist/bin/cli.js')
+    const child = spawn(process.execPath, [file])
+
+    const timeout = setTimeout(() => {
+      child.kill('SIGINT')
+    }, 120000)
```

Comment:
Really great PR! 😄 

From my experiences with client runs these timeouts might not be enough to get reliable results, so for now we might want to go up here to 180000 or even 240000 (for CI this should be still ok).

Reply:
increased in https://github.com/ethereumjs/ethereumjs-client/pull/176/commits/b2f29c845d8b722b4aee36db94937cd43451d17d